packages/alfa-dom/test/get-assigned-nodes.spec.tsx

```diff
@@ -2,16 +2,77 @@ import { jsx } from "@siteimprove/alfa-jsx";
 import { test } from "@siteimprove/alfa-test";
 import { getAssignedNodes } from "../src/get-assigned-nodes";
 
+const slot = <slot name="foo" />;
+const slot2 = <slot name="bar" />;
```

Comment:
50 lines down I will have forgotten the difference between `slot` and `slot2` and will need to scroll back to the top of the file to make sense of test cases 😉